multiversx_sdk/account_management/account_controller.py

```diff
@@ -14,12 +16,20 @@ def __init__(self, chain_id: str) -> None:
     def create_transaction_for_saving_key_value(self,
                                                 sender: IAccount,
                                                 nonce: int,
-                                                key_value_pairs: dict[bytes, bytes]) -> Transaction:
+                                                key_value_pairs: dict[bytes, bytes],
+                                                guardian: Optional[Address] = None,
+                                                relayer: Optional[Address] = None) -> Transaction:
         transaction = self.factory.create_transaction_for_saving_key_value(
             sender=sender.address,
             key_value_pairs=key_value_pairs
         )
 
+        if guardian:
+            transaction.guardian = guardian
+
+        if relayer:
+            transaction.relayer = relayer
```

Comment:
On `Transaction`, they are already defined as optional - maybe simply set, without check?

Reply:
that is indeed right.